lib/util.js

```diff
@@ -923,14 +923,16 @@ function formatWeakSet(ctx, value, recurseTimes, keys) {
 
 function formatWeakMap(ctx, value, recurseTimes, keys) {
   const maxArrayLength = Math.max(ctx.maxArrayLength, 0);
-  const entries = previewEntries(value).slice(0, maxArrayLength + 1);
-  const remainder = entries.length > maxArrayLength;
-  const len = entries.length - (remainder ? 1 : 0);
+  const [ entries ] = previewEntries(value).slice(0, (maxArrayLength + 1) * 2);
```

Comment:
nit: we don't usually put spaces inside of `[]`. `[entries]` is fine.

Reply:
The slice is actually unnecessary overhead. The limitation is done below and this has no effect (there are just two entries returned).